src/Tribe/RSVP.php

```diff
@@ -2049,32 +2086,35 @@ public function get_attendee( $attendee, $post_id = 0 ) {
 			'user_id'            => $user_id,
 			'ticket_sent'        => $ticket_sent,
 
-			// Fields for Email Tickets
-			'event_id'      => get_post_meta( $attendee->ID, self::ATTENDEE_EVENT_KEY, true ),
-			'ticket_name'   => ! empty( $product ) ? $product->post_title : false,
-			'holder_name'   => get_post_meta( $attendee->ID, $this->full_name, true ),
-			'holder_email'  => get_post_meta( $attendee->ID, $this->email, true ),
-			'order_id'      => $attendee->ID,
-			'ticket_id'     => $ticket_unique_id,
-			'qr_ticket_id'  => $attendee->ID,
-			'security_code' => $security,
-
-			// Attendee Meta
-			'attendee_meta' => $meta,
-		) );
+			// Fields for Email Tickets.
+			'event_id'           => get_post_meta( $attendee->ID, self::ATTENDEE_EVENT_KEY, true ),
+			'ticket_name'        => ! empty( $product ) ? $product->post_title : false,
+			'holder_name'        => get_post_meta( $attendee->ID, $this->full_name, true ),
+			'holder_email'       => get_post_meta( $attendee->ID, $this->email, true ),
+			'order_id'           => $attendee->ID,
+			'ticket_id'          => $ticket_unique_id,
+			'qr_ticket_id'       => $attendee->ID,
+			'security_code'      => $security,
+
+			// Attendee Meta.
+			'attendee_meta'      => $meta,
+
+			// Handle initial Attendee flags.
+			'is_subscribed'      => tribe_is_truthy( get_post_meta( $attendee->ID, $this->attendee_subscribed, true ) ),
+			'is_purchaser'       => true,
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 338aa41fff6862e26c26860473acc936</sub>